terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -742,13 +742,24 @@ def remove_inconsequential_statements(statements: List[JSON]) -> List[JSON]:
                 ]
             }
         },
+        "aws_s3_bucket": {
+            "gitlab": {
+                "bucket": "azul-gitlab-lb",
```

Comment:
You'll need to add this bucket name within azul.config. S3 bucket namespaces are global, so if we use this name in `dev` we cant use the same name in `prod`

Reply:
Configured the bucket named as recorded in PL, let me know if it looks OK.

Also added the requested [FIXME](https://github.com/DataBiosphere/azul/issues/3620).